### DISCUSSION TITLE
Can't pass unstable flags to tasks

### DISCUSSION THREAD

**Discussion by lishaduck** [denoland/deno#27120] [Q&A] [answered]
I'm trying to write tests using Vitest for `@lishaduck/effect-platform-deno`. I use `deno task` to run the invocations, but I can't pass unstable flags. I know I could use `deno test`, but I want to eventually upstream it into Effect, which uses Vitest.
I've tried `deno --unstable-kv task test`, which failed with "unknown option" and `deno task --unstable-kv test`, which worked, but didn't pass the flag on.
`deno --unstable-kv ./node_modules/.bin/vitest run` worked, but I feel that there's probably a better way to do it I'm just missing (If not, consider this a feature request).

**Comment by lishaduck** [comment]
If anyone stumbles upon this, I realized you can use the `unstable` key in `deno.json`. Not exactly what I was looking for, but it works well enough.

**Comment by BlackAsLight** [comment, marked as answer]
Put the flag at the end and it will be passed to the end of the task command

**Reply by lishaduck** [reply]
🤦‍♂️
